### PR TITLE
Update code to php 7.0

### PR DESCRIPTION
**Pull request by nickvergessen**
- [x] Conflict with #112

### CODE REVIEW

File: appinfo/info.xml [hunk 1]
```diff
@@ -13,19 +15,21 @@
 
 📣 Announcement center: An announcement was posted by an admin]]></description>
 
-	<licence>AGPL</licence>
-	<author>Joas Schilling</author>
 	<version>2.2.1</version>
+	<licence>agpl</licence>
+	<author>Joas Schilling</author>
+
+	<default_enable/>
```
Comment (MorrisJobke) [inline appinfo/info.xml]: This is not really needed. We use the shipped.json from the server repo

Comment (nickvergessen) [reply appinfo/info.xml]: default enabled is not equal to "shipped" and not equal to "always enabled". there is no information in shipped.json for "default enabled".
And yes, the value is read from info.xml
https://github.com/nextcloud/server/blob/9dc3b0480228a3c35b3b7a7cb1a904006f7e0762/lib/private/Installer.php#L521

**Comment by nickvergessen** [comment]
Tests pass, merging

File: appinfo/info.xml (hunk 1 above)
Comment (MorrisJobke) [reply appinfo/info.xml]: Ah .. .yes mixed it up. Sorry.

**Pull request merged**